llvm/test/CodeGen/AIE/aie2/schedule/interblock/liveness.mir

```diff
@@ -0,0 +1,228 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+# RUN: llc --mtriple=aie2 --run-pass=postmisched --issue-limit=1 -debug-only=machine-scheduler %s -o - 2>%t.log
+# RUN: cat %t.log | FileCheck %s --check-prefix=CHECK-LIVENESS
+# REQUIRES: asserts
+
+# These tests verify liveouts of every MachineBasicBlock in MachineFunction
+
+#     bb.0
+#     /  \
+# bb.1    bb.2
+#     \  /
+#     bb.3
+
+---
```

Comment:
I think some short description for each test would help. I think here what is important is that `$srcarry` is found not to be live, because no-one reads it in the function.